hack/.ci/run-e2e-gke.sh

```diff
@@ -21,14 +21,15 @@ SO_NODECONFIG_PATH="${SO_NODECONFIG_PATH=./hack/.ci/manifests/cluster/nodeconfig
 export SO_NODECONFIG_PATH
 SO_CSI_DRIVER_PATH="${parent_dir}/manifests/namespaces/local-csi-driver/"
 export SO_CSI_DRIVER_PATH
+SO_SCYLLACLUSTER_STORAGECLASS_NAME="${SO_SCYLLACLUSTER_STORAGECLASS_NAME=}"
+export SO_SCYLLACLUSTER_STORAGECLASS_NAME
 
 # Backwards compatibility. Remove when release repo stops using SO_DISABLE_NODECONFIG.
 if [[ "${SO_DISABLE_NODECONFIG:-false}" == "true" ]]; then
   SO_NODECONFIG_PATH=""
   SO_CSI_DRIVER_PATH=""
 else
-  # Make sure there is no default storage class before we create our own.
-  unset-default-storageclass
+  SO_SCYLLACLUSTER_STORAGECLASS_NAME="${SO_SCYLLACLUSTER_STORAGECLASS_NAME:-scylladb-local-xfs}"
```

Comment:
same here

Reply:
I think generally just defaulting this to `scylladb-local-xfs` would be fine and the caller can always choose to set it explicitly

Reply:
> I think generally just defaulting this to scylladb-local-xfs would be fine and the caller can always choose to set it explicitly

I ended up setting it to an empty string if SO_DISABLE_NODECONFIG is true for ease of debugging - in cases like the serial job you have to set `SO_SCYLLACLUSTER_STORAGECLASS_NAME` explicitly, otherwise you end up with an empty storageClassName, which is easier to spot (https://gcsweb.scylla-operator.scylladb.com/gcs/scylla-operator-prow/pr-logs/pull/scylladb_scylla-operator/2009/pull-scylla-operator-master-e2e-gke-serial/1810962794783182848/artifacts/deploy/0/manager/50_scyllacluster.yaml). It's a minor improvement though, so if you think simplifying the default more valuable, I can change it.

Reply:
done